src/main/java/xyz/groundx/caver_ext_kas/kas/wallet/Wallet.java

```diff
@@ -1929,4 +2030,67 @@ AccountUpdateKey makeUncompressedKeyFormat(AccountUpdateKey updateKey) {
 
         return updateKey;
     }
+
+    /**
+     * Validate whether given migrationAccount is valid or not.
+     * @param migrationAccount An account to be migrated to KAS Wallet
+     * @return boolean
+     */
+    private boolean validateMigrationAccount(MigrationAccount migrationAccount) {
+        if(migrationAccount.getAddress() == "") {
+            System.out.println("ERROR: Address of migrationAccount must not be empty.");
+            return false;
+        }
+
+        MigrationAccountKey<?> migrationAccountKey = migrationAccount.getMigrationAccountKey();
+        if(migrationAccountKey == null) {
+            System.out.println("ERROR: MigrationAccountKey of migrationAccount must not be empty.");
+            return false;
+        }
+
+        String keyClassName = migrationAccountKey.getClass().getSimpleName();
+
+        if(
+                keyClassName.equals(SinglePrivateKey.class.getSimpleName()) == false
```

Comment:
Do we need to know keyClassName? how about use a instanceOf keyword?

Reply:
Great idea. I will apply your feedback soon.

Reply:
@sirano11 
I applied your feedback [here](https://github.com/ground-x/caver-java-ext-kas/pull/80/commits/dd778a6e2b681e829f1a0081a72c4fde20a72803) :)

Now checking logic uses `instanceof` method.